src/common/core/core.js

```diff
@@ -1659,6 +1663,173 @@ define([
+         *
+         * @func
+         */
+        this.getLibraryNames = core.getLibraryNames;
+
+        /**
+         * Return the root of the inheritance chain of your Meta nodes.
+         *
+         * @param {module:Core~Node} node - any node in your project.
+         *
+         * @return {module:Core~Node} - Returns the acting FCO of your project.
+         *
+         * @func
+         */
+        this.getFCO = core.getFCO;
+
+        /**
+         * Returns if the node in question is a library root or not.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {bool} - Returns true if your node is a library root (even if it is embedded in other library),
+         * false otherwise.
+         *
+         * @func
+         */
+        this.isLibraryRoot = core.isLibraryRoot;
+
+        /**
+         * Returns if the node in question is a library element or not.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {bool} - Returns true if your node is a library element, false otherwise.
+         *
+         * @func
+         */
+        this.isLibraryElement = core.isLibraryElement;
+
+        /**
+         * Returns the fully qualified name of any node, which is the list of its namespaces separated
+         * by comma and followed by the name of the node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {string} - Returns the fully qualified name of the node (like: 'library.myName').
+         *
+         * @func
+         */
+        this.getFullyQualifiedName = core.getFullyQualifiedName;
+
+        /**
+         * Removes a library from your project. It will also remove any remaining instances of the specific library.
+         *
+         * @param {module:Core~Node} node - any node in your project.
+         * @param {string} name - the name of your library.
+         *
+         * @func
+         */
+        this.removeLibrary = core.removeLibrary;
+
+        /**
+         * Returns the origin GUID of any library node.
+         *
+         * @param {module:Core~Node} node - the node in question.
+         *
+         * @return {GUID | null} - Returns the origin GUID of the node or null if the node is not a library element.
+         *
+         * @func
+         */
+        this.getLibraryGuid = core.getLibraryGuid;
+
+        /**
+         * Rename a library in your project.
+         *
+         * @param {module:Core~Node} node - any node in your project.
+         * @param {string} oldName - the current name of the library.
+         * @param {string} newName - the new name of the project.
+         *
+         * @func
+         */
+        this.renameLibrary = core.renameLibrary;
+
+        /**
+         * Returns the info associated with the library.
+         *
+         * @param {module:Core~Node} node - any node in the project.
```

Comment:
this just needs any node, so it can lookup the library info, the only 'meaningful' input here is the name of the library